projects/span_suggesters/configs/chunk_suggester.cfg

```diff
@@ -0,0 +1,151 @@
+[paths]
+train = null
+dev = null
+vectors = null
+init_tok2vec = null
+span_key = null
```

Comment:
I think this should be consistently called `spans_key` and it shouldn't be under the `[paths]` section because that will be confusing for users. Even if it's shared between multiple components, you can define it in one spot (in spancat?) and then refer to that config variable in any other section, just like we do for tok2vec widths in the default configs:

```ini[components.tok2vec.model.embed]
@architectures = "spacy.MultiHashEmbed.v2"
width = ${components.tok2vec.model.encode.width}
attrs = ["NORM","PREFIX","SUFFIX","SHAPE"]
rows = [5000,2500,2500,2500]
include_static_vectors = false

[components.tok2vec.model.encode]
@architectures = "spacy.MaxoutWindowEncoder.v2"
width = 96
depth = 4
window_size = 1
maxout_pieces = 3
```